kpconv_torch/io/ply.py

```diff
@@ -234,7 +161,10 @@ def write_ply(filename, field_list, field_names, triangular_faces=None):
     # check all fields have the same number of data
     n_points = [field.shape[0] for field in field_list]
     if not np.all(np.equal(n_points, n_points[0])):
-        print("wrong field dimensions")
+        print(
+            f"wrong field dimensions: all fields should have {n_points[0]} \
+                points, some have {np.min(n_points)}."
```

Comment:
```suggestion
            f"wrong field dimensions: all fields should have {n_points[0]} "
            f"points, some have {np.min(n_points)}."
```

(Avoiding the `\` line escape.)